README.md

```diff
@@ -1,21 +1,105 @@
 # osl-rt
 
-TODO: Enter the cookbook description here.
-
 ## Requirements
 
 ### Platforms
 
-- CentOS 7+
+- Almalinux 8+
 
 ### Cookbooks
 
 ## Attributes
 
+Do **NOT** set the configuration in the attributes, instead use data bags.
+
+Name       | Type   | Description                                                            | Default
+-----------|--------|------------------------------------------------------------------------|---------
+`data-bag` | String | The name of the databag item. The data bag is always `request-tracker` | nil
+
+## Data Bag Attributes
+
+Name             | Type   | Description                                                  | Default
+-----------------|--------|--------------------------------------------------------------|----------
+`db-username`    | String | The username of the DB user                                  | nil
+`db-password`    | String | The password of the DB user                                  | nil
+`root-password`  | String | The password used for the root account on RT                 | nil
+`fqdn`           | String | The FQDN of the site and email                               | `example.org`
+`user`           | String | The user account that is responsible for being the default email | `support`
+`internal-domain`| String | A workaround required needs a non-sublevel domain name to access the site internally | `rtlocal`
+`db.type`        | String | The database type, MySQL or Postgres                         | `mysql`
+`db.host`        | String | The hostname of the DB server                                | `localhost`
+`db.name`        | String | The DB name on the DB server                                 | `rt`
+`queues`         | Hash   | The queues and emails available for RT. The key is the pretty print, and the value is a email-valid name. | `{'Support Example': nil}` Any nil-valued key will be ignored.
+`plugins`        | Array  | A list of [plugins](https://rt-wiki.bestpractical.com/wiki/Extensions) to add to the RT site | `[]`
+`lifecycles`     | Hash   | `WARNING: Default set to a custom lifecycle, explicitly set this to nil for actual default.` Any [custom lifecycles](https://docs.bestpractical.com/rt/4.4.1/customizing/lifecycles.html) to make available in RT | A custom example of a default lifecycle.
+
+### Example Data Bag Attributes
+
+```json
+# Create three different Queues. The keys are the "pretty print" name, while the values are the email name.
```

Comment:
JSON doesn't have comments and this makes it hard to read. Can you please just show the whole databag as an example?